tests/files/packet_debian10-containerd-migrate-crio.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# Instance settings
+cloud_image: debian-10
+mode: default
+
+# Docker specific settings:
+container_manager: containerd
+etcd_deployment_type: docker
```

Comment:
This only works for `container_manager=docker`, I think these tests should use `etcd_deployment_type=host` so the container manager upgrade does not affect the run. Note that the testing logic does not quite account for changing `container_manager` between runs so a new logic (script) may need to be introduced.